index.html

```diff
@@ -0,0 +1,176 @@
+					 <span class="point"></span>
+					 <li class="backend">Back End Dev</li>
+					 <span class="point"></span>
+					 <li class="year">2015</li>
+			  </ul>
+			   <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			   
+			   <ul class="skills">
+				 <li class="htl"><a href="#"></a>html</li>
+				 <li class="css"><a href="#"></a>css</li>
+				 <li class="js"><a href="#"></a>javascript</li>
+			   </ul>
+			   <button class="seeproject">See Project</button>
+			 </div>
+		 </div>
+	 
+		 <div class="container2">
+			 <img class="snapshoot" src="Images/Snapshoot2.png" alt="mobile image">
+			 <div class="daily">
+			   <h2 class="Tonic2">Multi-Post Stories</h2>
+			   <ul class="offer">
+					  <li class="canopy">canopy</li>
+					  <span class="point"></span>
+					  <li class="backend">Back End Dev</li>
+					  <span class="point"></span>
+					  <li class="year">2015</li>
+			   </ul> 
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="skills">
+				  <li class="htl"><a href="#"></a>html</li>
+				  <li class="css"><a href="#"></a>css</li>
+				  <li class="js"><a href="#"></a>javascript</li>
+				</ul>
+			   <button class="seeproject">See Project</button>
+			  </div>
+		  </div>
+	 
+		  <div class="container3">
+			 <img class="snapshoot" src="Images/Snapshoot3.png" alt="mobile image">
+			 <div class="daily">
+			   <h2 class="Tonic">Tonic</h2>
+			   <ul class="offer">
+					  <li class="canopy">canopy</li>
+					  <span class="point"></span>
+					  <li class="backend">Back End Dev</li>
+					  <span class="point"></span>
+					  <li class="year">2015</li>
+			   </ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="skills">
+				  <li class="htl"><a href="#"></a>html</li>
+				  <li class="css"><a href="#"></a>css</li>
+				  <li class="js"><a href="#"></a>javascript</li>
+				</ul> 
+			   <button class="seeproject">See Project</button>
+			  </div>
+		  </div>
+	 
+		  <div class="container4">
+			 <img class="snapshoot" src="Images/Snapshoot4.png" alt="mobile image">
+			 <div class="daily">
+			   <h2 class="Tonic2">Multi-Post Stories </h2>
+			   <ul class="offer">
+					  <li class="canopy">canopy</li>
+					  <span class="point"></span>
+					  <li class="backend">Back End Dev</li>
+					  <span class="point"></span>
+					  <li class="year">2015</li>
+			   </ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="skills">
+				  <li class="htl"><a href="#"></a>html</li>
+				  <li class="css"><a href="#"></a>css</li>
+				  <li class="js"><a href="#"></a>javascript</li>
+				</ul> 
+			   <button class="seeproject">See Project</button>
+			  </div>
+		  </div>
+		 </div>
+	 
+	   <div class="about-container">
```

Comment:
Please use a `section` tag here instead. The `about` section is a major component of your portfolio so please use the section tag.